### DISCUSSION TITLE
Change log file location

### DISCUSSION THREAD

**Discussion by dpo** [Homebrew/discussions#2824] [Linux] [answered]
It's possible to change the cache location by setting the `HOMEBREW_CACHE` environment variable, but is it possible to change the location where log files are written?

Thanks.

**Comment by dpo** [comment, marked as answer]
`HOMEBREW_LOGS`. Nevermind.

**Reply by neutric** [reply]
Please mark this as answered. This helps others with a similar question filter available threads. It also prevents Homebrew contributors from spending time reading this, only to find that help is no longer needed.
